images/kubernetes-csi-external-provisioner/configs/latest.apko.yaml

```diff
@@ -0,0 +1,31 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - kubernetes-csi-external-provisioner
+    - wolfi-baselayout
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  run-as: root # image is not working with nonroot user
```

Comment:
This is something to be aware of, I've not yet been able to run as `nonroot`

Reply:
I think this is ok for this, and the other csi related things that need to mount to a host socket

Reply:
thanks